src/components/AccountModule/AccountModuleConnectedScreen.js

```diff
@@ -22,6 +22,9 @@ import {
   useWalletConnectionDetails,
 } from './connection-hooks'
 import WalletSyncedInfo from './WalletSyncedInfo'
+import { identifyUser, trackEvent, events } from '../../analytics'
+
+let userIdentified = false
 
```

Comment:
see if you can get rid of this global variable by detecting the `connected` event in AccountModule.js and trigger identifyUser() there.